ts/features/bonus/cgn/screens/merchants/CgnMerchantsListByCategory.tsx

```diff
@@ -47,13 +51,19 @@ const styles = StyleSheet.create({
 
 const CgnMerchantsListByCategory = () => {
   const dispatch = useIODispatch();
-  const currentCategory = useIOSelector(cgnSelectedCategorySelector);
```

Comment:
@emiliopavia I preferred to avoid a complete refactoring of the redux state and leaving the PR scope only to the main feature integration, maybe we can handle the removal of the state selector and so on in a dedicated story what do you think?